src/main/java/seedu/address/ui/MoreDetailsPanel.java

```diff
@@ -28,9 +28,6 @@
     private static final String FXML = "MoreDetailsPanel.fxml";
     private static final String DEFAULT_LABEL = "<No student selected>";
     private static final String DEFAULT_TEXT = "<No note found>";
-    private static final String DEFAULT_STYLE = "-fx-font-size: 11pt;\n" + "-fx-font-family: \"Segoe UI Semibold\";\n"
```

Comment:
This entire file, don't overwrite any changes. Just add in your part but dont remove lines like this^. Don't remove argument from display, don't remove the extracted methods, don't remove the //@@author tags. Put your changes in a method showAttendance() <- probably should be int showAttendance(int rows) <- use the rows to calculate the row of your labels. Then call showAttendance() at line 122.